admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -47,19 +47,19 @@ export const ApplicationPage: React.FunctionComponent = () => {
         setShowModal(false);
         const mutate = newEvent.mutate;
         mutate({
-            id: eventType.id ?? '',
+            id: eventType.id,
```

Comment:
If the id is undefined, keep it that way. We want to be able to quickly identify if we are updating or creating